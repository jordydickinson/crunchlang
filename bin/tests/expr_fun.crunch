fun add(x: int64, y: int64): int64 = x + y;
