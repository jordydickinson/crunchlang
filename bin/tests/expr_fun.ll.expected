; ModuleID = 'expr_fun.crunch'
source_filename = "expr_fun.crunch"

@llvm.global_ctors = appending global [1 x void ()*] [void ()* @init.ctors]

define i64 @"add$pure"(i64 %x, i64 %y) {
entry:
  %iaddtmp = add i64 %x, %y
  ret i64 %iaddtmp
}

define void @init.ctors() {
entry:
  ret void
}
